### PR TITLE
Update /web/css/system-color/index.md

### PR DESCRIPTION
**Pull request by lidejing**
### Related issues and pull requests
Non-standard system color extensions [#26596](https://github.com/mdn/content/issues/26596)

### CODE REVIEW

File: files/en-us/web/css/system-color/index.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Firefox also supports some non-standard extensions to the system color keyword s
 - `-moz-ButtonHoverFace` {{non-standard_inline}}
   - : The background color of a button that the mouse pointer is over (which would be `ThreeDFace` or `ButtonFace` when the mouse pointer is not over it). Should be used with the `-moz-ButtonHoverText` foreground color.
 - `-moz-ButtonHoverText` {{non-standard_inline}}
-  - : The text color of a button that the mouse pointer is over (which would be ButtonText when the mouse pointer is not over it). Should be used with the `-moz-ButtonHoverFace background` color.
+  - : The text color of a button that the mouse pointer is over (which would be ButtonText when the mouse pointer is not over it). Should be used with the `-moz-ButtonHoverFace` background color.
 - `-moz-CellHighlight` {{non-standard_inline}}
   - : Background color for selected item in a tree widget. Should be used with the `-moz-CellHighlightText` foreground color. See also `-moz-html-CellHighlight`.
 - `-moz-CellHighlightText` {{non-standard_inline}}
```
Comment (yarusome) [inline files/en-us/web/css/system-color/index.md:122]: The line right below this one also needs fixing

**Review by yarusome — COMMENTED** [review]
Thanks for taking this up! You could type `Fixes #26596` under "Related issues and pull requests" to make GitHub link this PR with the issue.

File: files/en-us/web/css/system-color/index.md [hunk 2]
```diff
@@ -129,7 +129,9 @@ Firefox also supports some non-standard extensions to the system color keyword s
   - : Background color for dialog boxes. Should be used with the `-moz-DialogText` foreground color.
 - `-moz-DialogText` {{non-standard_inline}}
   - : Text color for dialog boxes. Should be used with the `-moz-Dialog` background color.
-- `-moz-dragtargetzone` {{non-standard_inline}}, `-moz-EvenTreeRow` {{non-standard_inline}}
+- `-moz-dragtargetzone` {{non-standard_inline}}
+  - : Specify the areas where an element can be dropped during drag and drop operations.
```
[the remark this answers is not in the document]
Comment (momobutong) [reply files/en-us/web/css/system-color/index.md:133]: > Could you add a link (here in the review) to support this description? That would be quite helpful.

Of course. I will do it.

[the remark this answers is not in the document]
Comment (momobutong) [reply files/en-us/web/css/system-color/index.md:133]: > Could you add a link (here in the review) to support this description? That would be quite helpful.

I descript it by myself. There are some reference documents.
[line 1306 in https://searchfox.org/mozilla-central/source/__GENERATED__/layout/style/ServoStyleConsts.h#1307](https://searchfox.org/mozilla-central/source/__GENERATED__/layout/style/ServoStyleConsts.h#1307)

 in [http://help.dottoro.com/lanifqvh.php](http://help.dottoro.com/lanifqvh.php)

Comment (yarusome) [inline files/en-us/web/css/system-color/index.md:133]: :+1: Just tweak the wording per the code comment.
```suggestion
  - : Highlight color for regions where an element can be dropped during a drag-and-drop operation.
```

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to the team!

**Pull request merged**